python/src/scipp/plot/controller.py

```diff
@@ -0,0 +1,468 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+# @author Neil Vaytet
+
+from .tools import check_log_limits
+from .._utils import name_with_unit, value_to_string
+from .._scipp import core as sc
+import numpy as np
+
+
+class PlotController:
```

Comment:
Difficult to establish what the exact remit of this base class in WRT the derived types that utilise the functionality. Maybe a comment for the class could be provided to help clarify what it's responsibilities are.

Reply:
yes, docstrings needed everywhere. see above